### ISSUE TITLE
Progress bar showing series loading progress and thumbnails not showing 

### ISSUE THREAD

**Issue by ranasrule** [OHIF/Viewers#682]
In version 1.x of OHIF viewer a progress bar would show at the bottom of the thumbnail for each series in a study but this is absent in version 2. Also the thumbnail for each study just shows the text "loading" until the whole study is loaded whereas before in version 1.x they would appear right away as soon as the study viewer page opened.

**Comment by swederik** [comment]
Yeah when we originally start moving to React I implemented storing the progress data in Redux, but it was impacting performance so I disabled it for now. We should put this data in some container component instead or throttle the updates to the redux store. See https://github.com/OHIF/Viewers/blob/a3405e7a9ebea018690805b36790f97c557fbdc5/src/components/StudyLoadingMonitor.js
